src/Subscription/Subscription/ChangeLog.md

```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 0.7.4
+* Added new cmdlets `New-AzSubscriptionAlias`
```

Comment:
The description item of the change should be under upcoming release

Reply:
Fixed